pkg/driver/driver.go

```diff
@@ -39,3 +48,19 @@ func (d *BaseDriver) Start(_ context.Context) (chan error, error) {
 func (d *BaseDriver) Stop(_ context.Context) error {
 	return nil
 }
+
+func (d *BaseDriver) CreateSnapshot(_ context.Context, _ string) error {
+	return fmt.Errorf("unimplemented")
+}
+
+func (d *BaseDriver) ApplySnapshot(_ context.Context, _ string) error {
+	return fmt.Errorf("unimplemented")
+}
+
+func (d *BaseDriver) DeleteSnapshot(_ context.Context, _ string) error {
+	return fmt.Errorf("unimplemented")
+}
+
+func (d *BaseDriver) ListSnapshots(_ context.Context) (string, error) {
+	return "", nil
```

Comment:
Make sense, but would apply to all the commands in that case ?

Here it just returns "an empty list" (as string)

Reply:
Other commands were already returning error if am not wrong.
I would prefer error even though returning an empty list as string is not wrong.

Mainly because any of snapshot command should not work for vz driver.